src/schema/rules/sidecars/derivatives/common_derivatives.yaml

```diff
@@ -0,0 +1,85 @@
+---
+CommonDerivativeFields:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  fields:
+    Description: RECOMMENDED
+    Sources: OPTIONAL
+    RawSources: OPTIONAL
+
+SpatialReferenceEntity:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - entities contains "space"
+  fields:
+    SpatialReference: RECOMMENDED
+
+SpatialReferenceNonStandard:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - not(entities.space in schema.objects.metadata._StandardTemplateCoordSys)
+  fields:
+    SpatialReference: REQUIRED
+
+SpatialReferenceNoEntity:
+  selectors:
+  - dataset.DatasetType == "derivative"
+  - not(entities contains "space")
+  fields:
+    SpatialReference: REQUIRED
```

Comment:
Might be an inconsistency in the spec. I believe that a missing "space" entity is assumed to be `space-orig`.